registry/storage/validatorstore_test.go

```diff
@@ -339,3 +342,98 @@ func TestSelfValidatorStore_NilOperatorID(t *testing.T) {
 	require.Nil(t, selfStore.SelfParticipatingCommittees(99))
 	require.Nil(t, selfStore.SelfParticipatingCommittees(201))
 }
+
+func BenchmarkValidatorStore_Update(b *testing.B) {
+	shares := map[spectypes.ValidatorPK]*ssvtypes.SSVShare{}
+
+	const (
+		totalOperators  = 500
+		totalValidators = 50_000
+	)
+
+	var validatorIndex atomic.Int64
+	createShare := func(operators []spectypes.OperatorID) *ssvtypes.SSVShare {
+		index := validatorIndex.Add(1)
+
+		var pk spectypes.ValidatorPK
+		binary.LittleEndian.PutUint64(pk[:], uint64(index))
+
+		var committee []*spectypes.ShareMember
+		for _, signer := range operators {
+			committee = append(committee, &spectypes.ShareMember{Signer: signer})
+		}
+
+		return &ssvtypes.SSVShare{
+			Metadata: ssvtypes.Metadata{
+				BeaconMetadata: &beaconprotocol.ValidatorMetadata{
+					Index: phase0.ValidatorIndex(index),
+				},
+			},
+			Share: spectypes.Share{
+				ValidatorIndex:      phase0.ValidatorIndex(index),
+				ValidatorPubKey:     pk,
+				SharePubKey:         pk[:],
+				Committee:           committee,
+				FeeRecipientAddress: [20]byte{10, 20, 30},
+				Graffiti:            []byte("example"),
+			},
+		}
+	}
+
+	for i := 0; i < totalValidators; i++ {
+		committee := make([]spectypes.OperatorID, 4)
+		if rand.Float64() < 0.02 {
+			// 2% chance of a purely random committee.
+			for i, id := range rand.Perm(totalOperators)[:4] {
+				committee[i] = spectypes.OperatorID(id)
+			}
+		} else {
+			// 98% chance to form big committees.
+			first := rand.Intn(totalOperators * 0.2) // 20% of the operators.
+			for i := range committee {
+				committee[i] = spectypes.OperatorID((first + i) % totalOperators)
+			}
+		}
+		share := createShare(committee)
+		shares[share.ValidatorPubKey] = share
+	}
+
+	// Print table of committees and validator counts for debugging.
+	// committees := map[[4]spectypes.OperatorID]int{}
```

Comment:
Can we somehow keep it uncommented? `testing.B` doesn't have the `Verbose` method, but perhaps we could add a flag with `// nolint`

Reply:
i uncommented all except the Render line because i prefer not to spam the benchmark output unless the runner wants it (its a really big output)